### ISSUE TITLE
releases are not compatible with "go mod"

### ISSUE THREAD

**Issue by marcofranssen** [smartystreets/goconvey#551]
new go 11 mod feature does not support version which do not have the `v` prefix on version numbers.

So far I did not find a way to install goconvey using `go mod`
```go.mod
module github.com/marcofranssen/myproject

require github.com/smartystreets/goconvey/convey 1.6.3
```

In general other packages I depend on do work. 

```go.mod
module github.com/marcofranssen/myproject

require github.com/goreleaser/goreleaser v0.93.2
```

As you can see this package contians the v prefix. in the version number.

**Comment by amalhanaja** [comment]
did you changes yout `go.mod` file ?
I'm use this commands every add pacakges
`vgo get -u <package>` inside my work dir, it's inserted new line on my `go.mod` files

**Comment by nasermirzaei89** [comment]
I used:
```sh
go get -u github.com/smartystreets/goconvey
```

but added this line to my `go.mod`
```
github.com/smartystreets/assertions v0.0.0-20190215210624-980c5ac6f3ac // indirect
```

**Comment by vilakshminar** [comment]
I'm having the same issue. I think all the tags generated need to be prefixed with a `v` or else go mod will automatically replace the 1.6.3 with the commit hash in both the go.mod & go.sum files.<br>
It does work as it pulls the right commit but would be nice to make sure that go.mod has the version and not the commit.

**Comment by jpreese** [comment]
Was just going to ask the same thing. Would be nice if future tags could be prefixed with `v` for `go mod` support.

**Comment by marcofranssen** [comment]
Why not just tag the existing releases also with a go.mod compatible tag?
Should be doable imho.

**Comment by jpreese** [comment]
Agreed, that also works if the maintainers want to take that approach.

**Comment by nasermirzaei89** [comment]
no reply after 1 year from owners or maintainers. a library with more than 5k stars is more like a stale repository!!!

**Comment by jpreese** [comment]
There was a merge a few months ago, maybe they just got busy with other priorities :)

@riannucci do you have any thoughts on this?

**Comment by riannucci** [comment]
Hi!

(Sorry for not being more proactive maintainer-wise; I haven't gotten to do any development on goconvey for years now 😢)

If I understand correctly... All we need to do is to add vXXX tags for each XXX tag and go.mod should be happy with it?

Pretty sure I can do that... let me give it a shot.

R

**Comment by jpreese** [comment]
That's correct! The Go toolchain for modules explicitly looks for a `vX.Y.Z` tag when trying to resolve the latest version for a given repository.

**Comment by riannucci** [comment]
Okey dokey, I've created v versions of all existing tags. I'm going to try to create a new release as well, since it looks like the last one is pretty old.

**Comment by riannucci** [comment]
Alright, I've posted a v1.6.4 release as well, which contains everything currently on master.

**Comment by jpreese** [comment]
Awesome! Thanks so much

**Comment by riannucci** [comment]
Sorry for not doing this sooner!

Let me know if you run into any issues though, and I'll be happy to help get them landed.

One of these days I'll find some time to breathe some new life into goconvey...

**Comment by riannucci** [comment]
(marking this closed; please reopen if something's not working as expected)
